Adding updated connectors files (#9)
apollographql/rover-init-starters: start-with-rest/getting-started.md

3. Run this request in your terminal and compare it with what’s been configured using the `@connect` directive in `products.graphql`. You'll notice that some properties in the terminal response match to the `selection` mapping in the schema. This is the key to how Connectors work!

Hooray! You ran a query, got some data back, and reviewed what Connectors are like under the hood! Feel free to experiment some more–try tweaking the query to see what data you can retrieve. 🚀

# Time to build your API
You’re all set to start building. You'll be working primarily with the `products.graphql` file.

First, make sure you’ve installed and configured [your IDE extension of choice](https://www.apollographql.com/docs/graphos/schema-design/ide-support) so you can rely on its autocompletion, schema information, and syntax highlighting features.

Then, follow the development cycle below:

1. [Add your REST API details using @source](https://www.apollographql.com/docs/graphos/schema-design/connectors/directives#source). 
2. Define the types and fields you want your GraphQL API to expose. Use the inline comments labeled with a ✏️ to follow along.

> 3. [Configure the connector's request details](https://www.apollographql.com/docs/graphos/schema-design/connectors/requests).

Reviewer: ```suggestion
3. [Configure the Connector's request details](https://www.apollographql.com/docs/graphos/schema-design/connectors/requests).
```

> 4. [Configure the connector's response mapping](https://www.apollographql.com/docs/graphos/schema-design/connectors/responses). You can use the [Connectors Mapping Playground](https://www.apollographql.com/connectors-mapping-playground) to help convert JSON responses to and from GraphQL types.

Reviewer: ```suggestion
4. [Configure the Connector's response mapping](https://www.apollographql.com/docs/graphos/schema-design/connectors/responses). You can use the [Connectors Mapping Playground](https://www.apollographql.com/connectors-mapping-playground) to help convert JSON responses to and from GraphQL types.
```

5. Run operations and debug your API following the instructions in the section below.

📓 **Note:** If you’re working with APIs that require headers, you’ll need to include them in `products.graphql` and add a router configuration file (`router.yaml`) to your project directory.

To learn more about headers and other advanced features like configuring environment variables, telemetry, and authentication, visit [Apollo’s docs on working with Router](https://community.apollographql.com/c/graph-os/getting-started/35).

ℹ️ If you run into any issues or difficulties, please reach out via the [Apollo Community here](https://community.apollographql.com/c/graph-os/getting-started/35) and click “New Topic”–the Apollo team is here to help!

# Debugging your schema

> The Apollo dev toolkit includes a few debugging tools to help you design and develop your GraphQL API. The journey looks a little something like this:

Reviewer: ```suggestion
The Apollo dev toolkit includes a few debugging tools to help you design and develop your graph. The journey looks a little something like this:
```

- Design your schema with Apollo’s IDE extensions
- Check for errors each time you save
- Debug Connectors in Sandbox
- Rinse and repeat until you're happy with your API!

## Design your schema with Apollo’s IDE extensions

> Apollo’s IDE extensions are designed to help you catch and correct any issues related to schema design as early as possible. Lean on their instant feedback and autocomplete capabilities to help you create types, fields, arguments, and connectors.

Reviewer: ```suggestion
Apollo’s IDE extensions are designed to help you catch and correct any issues related to schema design as early as possible. Lean on their instant feedback and autocomplete capabilities to help you create types, fields, arguments, and Connectors.
```
